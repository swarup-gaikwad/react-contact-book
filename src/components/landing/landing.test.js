import React from "react";
import { shallow } from "enzyme";
import Landing from "./landing";

describe("Landing", () => {
  it("should render correctly", () => {
    const component = shallow(<Landing />);
  });
  it("should render initial layout", () => {
    // when
    const component = shallow(<Landing />);
    // then
    expect(component.getElements()).toMatchSnapshot();
  });
});
