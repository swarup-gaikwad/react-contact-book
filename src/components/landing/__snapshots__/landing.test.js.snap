// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render initial layout 1`] = `
Array [
  <div
    className="container"
  >
    <h1>
      Welcome to Contact Book1
    </h1>
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <img
          alt="conact book"
          src="../../../phone-book.png"
        />
      </div>
      <div
        className="col-md-6"
      >
        <h2>
          Lorem ipsum
        </h2>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </p>
      </div>
    </div>
  </div>,
]
`;
